Dockerfile

```diff
@@ -101,7 +101,7 @@ FROM builder AS testbuilder
 ARG DPSERVICE_FEATURES=""
 RUN meson setup release_build $DPSERVICE_FEATURES --buildtype=release && ninja -C release_build
 RUN CC=clang CXX=clang++ meson setup clang_build $DPSERVICE_FEATURES && ninja -C clang_build
-RUN meson setup xtratest_build $DPSERVICE_FEATURES -Denable_tests=true && ninja -C xtratest_build
```

Comment:
Is this still needed or just debugging leftover? The grpc wrapper python script should return all underlay addresses so that they can be saved for later and used, no need o have them static for the test suite usually.

Reply:
maglev hashing doesnt behave deterministic with shuffled underlay ipv6 addresses therefore it is still needed.